src/QueryStatisticsCalculator.php

```
@@ -29,14 +32,14 @@ public function calculate(array $queryResults): array
 
         $costs = array_column($queryResults, 'cost');
         $totalCount = count($costs);
-        $mean = $totalCount > 0 ? array_sum($costs) / $totalCount : 0;
+        $mean = $totalCount > 0 ? (float) array_sum($costs) / (float) $totalCount : 0.0;
```

PHPStan (2.1.33)

Comparison operation ">" between int<1, max> and 0 is always true.
Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your %configurationFile%.

(greater.alwaysTrue)